lib/models/MailPiece.dart

```diff
@@ -0,0 +1,10 @@
+class MailPiece {
+  int _mail_ID;
+  DateTime _informedDate;
+  String _LogoText;
+  AddressObject _senderAddress;
+
+   MailPiece( this._mail_ID, this._informedDate, this._LogoText, this._senderAddress );
+
+  
```

Comment:
You could probably remove these, and it wouldn't hurt to end the file in a newline.